omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -118,6 +112,8 @@ where
             && self.state == (Role::Follower, Phase::Accept)
             && self.handle_sequence_num(acc_ss.seq_num, acc_ss.n.pid) == MessageStatus::Expected
         {
+            // Don't have to handle flushed entries here because we will send Accepted after
```

Comment:
```suggestion
            // Flush entries before appending stopsign. The accepted index is ignored here as it will be updated when appending stopsign.
```